.github/workflows/ci.yaml

```diff
@@ -406,3 +406,28 @@ jobs:
         with:
           name: release-images
           path: ./release-images
+
+  trivy:
+    runs-on: ubuntu-18.04
+    needs: prebuild
+    steps:
+      - name: Checkout code
+      - uses: actions/checkout@v2
+
+      - name: Build an image from Dockerfile
+        run: |
+          docker build -t codercom/code-server:${{ github.sha }} ci/release-image/Dockerfile
```

Comment:
❓it's been a while since I've actually used `docker build`. I think this is right but not sure if I need `./` in front of the path?

Reply:
Actually. Now that I'm looking a bit closer. I wonder if I should instead run `sh ./ci/release-image/build.sh`? 🤔

https://github.com/cdr/code-server/blob/main/ci/release-image/build.sh#L8

And I also think my `docker build` syntax is off - missing `.` at end?

Reply:
And I think I need `-f` since I'm pointing to a file? 

Source: [Docker build docs](https://docs.docker.com/engine/reference/commandline/build/#options)